website/snippets/_new-sl-setup.md

```diff
@@ -67,26 +72,28 @@ dbt Cloud Enterprise plans can optionally add multiple credentials and map them
 
 We recommend configuring credentials and service tokens to reflect your teams and their roles. For example, create tokens or credentials that align with your team's needs, such as providing access to finance-related schemas to the Finance team.
 
-Note that: 
+<Expandable alt_header="Considerations for linking credentials">
+
 - Admins can link multiple service tokens to a single credential within a project, but each service token can only be linked to one credential per project.
 - When you send a request through the APIs, the service token of the linked credential will follow access policies of the underlying view and tables used to build your semantic layer requests.
 - <SLEnvVars/>
-
-To add multiple credentials and map them to service tokens:
-
-1. After configuring your environment, on the **Credentials & service tokens** page, click the **Add Semantic Layer credential** button to create multiple credentials and map them to a service token.
-2. In the **1. Add credentials** section, fill in the data platform's credential fields. We recommend using “read-only” credentials.
-<Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-add-credential.jpg" width="55%" title="Add credentials and map them to a service token. " />
-
-3. In the **2. Map new service token** section, map a service token to the credential you configured in the previous step. dbt Cloud automatically selects the service token permission set you need (Semantic Layer Only and Metadata Only).
-
-4. To add another service token during configuration, click **Add Service Token**. 
-5. You can link more service tokens to the same credential later on in the **Semantic Layer Configuration Details** page. To add another service token to an existing Semantic Layer configuration, click **Add service token** under the **Linked service tokens** section.
-6. Click **Save** to link the service token to the credential. Remember to copy and save the service token securely, as it won't be viewable again after generation.
+</Expandable>
+
+#### 1. Add more credentials
+- After configuring your environment, on the **Credentials & service tokens** page, click the **Add Semantic Layer credential** button to create multiple credentials and map them to a service token. <br />
+- In the **1. Add credentials** section, fill in the data platform's credential fields. We recommend using “read-only” credentials.
+   <Lightbox src="/img/docs/dbt-cloud/semantic-layer/sl-add-credential.jpg" width="55%" title="Add credentials and map them to a service token. " />
+
+#### 2. Map service tokens to credentials
+- In the **2. Map new service token** section, [map a service token to the credential](/docs/use-dbt-semantic-layer/setup-sl#map-service-tokens-to-credentials) you configured in the previous step. dbt Cloud automatically selects the service token permission set you need (Semantic Layer Only and Metadata Only).
```

Comment:
Hiya @mirnawong1 

Just to ask, should that number 2 be part of the first sentence?

Kind regards